_delphi_utils_python/delphi_utils/data/fips_hhs_table.csv

```diff
@@ -1,5 +1,4 @@
 fips,hhs
-01000,4
```

Comment:
Oh probably not, if they get coded to state and then to hhs. I wonder if it would be useful to put a warning in `replace_geocode` that fips -> hhs won't preserve megacounties.

Reply:
do they _always_ get coded to state first, or just in jhu/usafacts/combo?

i'm in favor of a warning

Reply:
is there a reason not to go with @dshemetov's original suggestion to adding them to this mapping? Not having them here was purely an oversight.

Reply:
i don't think so -- probably better to add them

Reply:
:+1: will do